odo/backends/sql.py

```diff
@@ -181,7 +181,7 @@ def getbind(t, bind):
     if bind is None:
         return t.bind
 
-    if isinstance(bind, sa.engine.base.Engine):
+    if isinstance(bind, sa.engine.interfaces.Connectable):
```

Comment:
Are there other connectables in here that should be explicitly wired?

Reply:
connectable is the interface, so anything that works here should be fine.